### DISCUSSION TITLE
Eliminating print statement clutter from Python script

### DISCUSSION THREAD

**Discussion by HenryWede** [isl-org/Open3D#6386] [Q&A]
Hello,
In a Python script, we are using the 

> o3d.pipelines.registration.registration_colored_icp

method to check if two scans have any overlapping areas.  Often the result is "no".  In those cases, there are print statements to the DOS window that are really cluttering things up.  I have tried using the open3d.utility.VerbosityLevel to stop them, but because they are considered errors it doesn't seem to work.

I tried to redirect the stdout from the Python script, but I think the printing is done from another module that I don't have any control over.  That is just a theory.

Can anyone please let me know how to stop the printing?

Thanks for your time.

**Comment by saurabheights** [comment]
Try discarding standard error as well.

`python3 test.py > /dev/null 2&>1`

![image](https://github.com/isl-org/Open3D/assets/11634769/a920b2cb-b9b8-4ed1-98af-e127e4c28120)

I did check if there is a way to disable logging, but unfortunately no success. Can only lower verbosity level to Error.
